### DISCUSSION TITLE
[useCalendar] visibleDuration and month change

### DISCUSSION THREAD

**Discussion by knoefel** [adobe/react-spectrum#4560] [Q&A] [answered]
Is it possible to render 2 months via `visibleDuration: { months: 2 }` and only move by one month, when changing the months via the navigation buttons. For example on initial render it will render:

May 2023 and June 2023

after pressing the next button i want it to display:

June 2023 and July 2023 (right now it will render July 2023 and August 2023)

I'm using the [Tailwind Rangepicker](https://codesandbox.io/s/objective-shape-8r4utm?file=/src/Calendar.js) example from the docs.

Sorry if this is a silly question, i searched the docs and github for quite some time now, but unfortunately without success.

Thanks in advance!!

For example, in react-day-picker it's possible to configure if you want to navigate the number of months per time when rendering multiple months.

https://react-day-picker.js.org/basics/customization#paged-navigation

**Comment by LFDanLu** [comment, marked as answer]
At the moment this isn't supported, but we are tracking this via https://github.com/adobe/react-spectrum/issues/3284.
